src/pages/Login/components/User.js

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import GlobalStyle from '../../../styles/GlobalStyle';
+import KakaoModal from './KakaoModal';
+import * as S from './User.styled';
+
+const User = ({ content }) => {
```

Comment:
```suggestion
const User = ({ content: { title, text, url, text2 } }) => {
```
구조분해할당을 위와 같은 방법으로 한번 더 진행해 줄 수 있습니다!